crates/matrix-sdk-ui/src/timeline/builder.rs

```diff
@@ -212,7 +212,7 @@ impl TimelineBuilder {
             let room_event_cache = room_event_cache.clone();
             let inner = controller.clone();
 
-            let span = info_span!(
+            let span = trace_span!(
```

Comment:
I think we should keep these spans at the regular log level, as they're important to realize where a log message is coming from.

Reply:
restored.